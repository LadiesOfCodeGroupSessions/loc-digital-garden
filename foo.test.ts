import { describe, expect, it } from 'vitest'

// Edit an assertion and save to see HMR in action

describe('Foo', () => {
      it('foo', () => {
          expect(Math.sqrt(4)).eq(2)
      }),
    it('snapshot', () => { expect({ foo: 'bar' }).toMatchSnapshot() })
})