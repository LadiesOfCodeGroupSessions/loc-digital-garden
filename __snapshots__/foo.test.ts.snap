// Vitest Snapshot v1

exports[`Foo > snapshot 1`] = `
{
  "foo": "bar",
}
`;
